.github/workflows/ocr-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: OCR Poetry Tests
+on:
+  pull_request:
+    types: [opened, reopened]
+    paths:
+      - .github/workflows/ocr-tests.yml
+      - OCR/**
```

Comment:
Do we want to make this scope even more narrow to just the `ocr` and `tests` directories within `OCR`?

Reply:
no because we would still want to run the for dependency changes for example, pyproject.toml lives outside these two folders
my vote is the entire OCR directory

Reply:
For sure! I think thats the safe bet for now. Also, this is more in regards to when this CI will trigger. so for example, if you make a change in the `pyproject.toml` file and commit to your feature branch it will trigger this workflow.